docs/theming.md

```diff
@@ -185,9 +185,170 @@ extension MyViewController {
 ```
 <!--</div>-->
 
+## Migration guide: Themers to Theming extensions
+
+Themers will eventually be deprecated and deleted. Theming extensions are the recommended
+replacement. This migration guide will cover the typical migration path from Themer usage to
+Theming extensions.
+
+In general, every component's `Themer` targets will gradually be replaced by a single `Theming`
+extension target. This includes:
+
+- ColorThemer
+- FontThemer
+- ShapeThemer
+- TypographyThemer
+- Component Themers (e.g. CardThemer)
+
+Some Theming extensions are still in Beta. To add a Beta Theming extension to your project you will
+need to follow the instructions in the [Beta components](../contributing/beta_components.md)
+documentation.
+
+Some components do not have a Theming extension yet. We are prioritizing the remaining Theming
+extensions in https://github.com/material-components/material-components-ios/issues/7172.
+
+#### Typical migration
+
+When migrating from a subsystem Themer to a Theming extension the first thing to understand is that
+a Theming extension will apply *all* of the available subsystems to a given component. If you were
+previously relying on the ability to apply only one subsystem (e.g. Color) to a component, please
+file a
+[feature request](https://github.com/material-components/material-components-ios/issues/new/choose)
+with a code snippet of your existing use case.
+
+Generally-speaking, the migration from a subsystem Themer to a Theming extension will involve the
+following steps:
+
+1. Update your dependencies.
+2. Update your imports.
+3. Changes to your code.
```

Comment:
What about the pre-implementation/planning work? Like: planning where in an app a Material container scheme should live so it can be shared with all material components.

Reply:
I'm thinking of leaving that to the reader; we do cover app architectures a bit in the Theming extensions documentation above. Do you feel we should give guidance here?